.appveyor.yml

```diff
@@ -20,6 +24,8 @@ clone_folder: *REPO_DIR
 cache:
   - *CACHE_DIR
 
+clone_depth: 50
```

Comment:
tried long ago, had some problems that the builds can't clone when PR is updated again before the build starts.